spec/abci/abci++_basic_concepts.md

```diff
@@ -363,7 +362,7 @@ Example:
 }
 ```
 
-## Evidence
+## Evidences of Misbehavior
```

Comment:
Below, lines 373 and 374: we are still confusing evidence and misbehavior.

Comet detects misbehavior, produces an evidence, broadcast it, nodes verify the evidence, evidences are included in blocks. Once (verified) evidences are committed, the associated misbehavior is notified to the application. We don't send evidences to the application, we send the misbehavior. The proof of misbehavior, i.e., evidence, is produced and verified by Comet.

Reply:
re-worded a bit, thanks for the explanation